test/test_build.sh

```diff
@@ -18,7 +19,10 @@ echo "Installing required dependencies for distribution '${1}'..."
 case $1 in
 	alpine)
 		apk update
-		apk add alpine-sdk autoconf automake bison check-dev clang flex gettext git libtool linux-headers sudo xz
+		apk add alpine-sdk autoconf automake bison check-dev clang flex gettext libtool linux-headers sudo xz
+		;;
+	fedora)
+		sudo dnf install -y autoconf automake bison check-devel clang flex gettext libtool make pkgconfig xz
```

Comment:
No `sudo dnf update`?    ( There  is `apk update` and `sudo apt update` )

Reply:
No, because dnf behavior is different from apt's one : `dnf update` is used to update the system (like `apt upgrade`). Packages list is automatically maintained by dnf (I learned this today !). See https://fedoraproject.org/wiki/Differences_to_Ubuntu#Package_Management for reference.